src/test/groovy/uk/gov/hmcts/dm/it/BaseIT.groovy

```diff
@@ -136,6 +148,12 @@ class BaseIT {
         token
     }
 
+    def serviceToken()
+    {
+        def s2sToken = authTokenProvider.findServiceToken()
+        s2sToken
```

Comment:
couldn't we remove the `s2sToken` token part?
and just return `authTokenProvider.findServiceToken()`